debian/changelog

```diff
@@ -1,8 +1,9 @@
 make-music (3.14.0-0) unstable; urgency=low
 
   * Applied changes from v3.9.2 es_AR backports to latest
+  * Adapted dependency to also build and install on Debian Stretch
```

Comment:
This should be in a new entry together with a version change, however the version is unknown yet.

Reply:
Good! I've done the same version bump for the home-button (linked PR)